src/requirements.txt

```diff
@@ -1,11 +1,10 @@
 # Standard packages needed for the docs
 ###############################################################################
 
-docutils==0.14
+docutils>=0.14
 sphinx-autobuild
 
 # These versions should match the Read The Docs environment (please update if
 # you notice this is not the case)
-alabaster>=0.7,<0.8,!=0.7.5
 setuptools<41
-sphinx==1.7.4
+sphinx>=1.8.5,<4
```

Comment:
shouldn't this be
```suggestion
sphinx>=1.8.5
```

Reply:
I believe Sphinx 3.x will stay compatible with everything which is currently out there in the wild. However, I am not sure how this will be with Sphinx 4.x. So: Better safe than sorry.

Reply:
hm ok valid point